app/models/manageiq/providers/openstack/cloud_manager/provision/cloning.rb

```diff
@@ -4,7 +4,7 @@ def do_clone_task_check(clone_task_ref)
       instance = openstack.handled_list(:servers).detect { |s| s.id == clone_task_ref }
       status   = instance.state.downcase.to_sym
 
-      return true if status == :active
+      return true if [:active, :error].include?(status)
```

Comment:
@gmcculloug Right, so raising the exception should move into on_error state of the state machine? I think that looks cleaner, then we will need a method inside on_error state, that will pick another network and retry the provisioning step. Can we do that?